.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,113 @@
+name: Bug Report
+description: Create a report to help us improve
+title: "[Bug]: "
+labels: ["bug", "support_request"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide all the necessary information to expedite the response.
+        #Start here
+  - type: input
+    id: ov_version
+    attributes:
+      label: OpenVINO Version 
+      description: OpenVINO Version or Branch/tag in OpenVINO Github
+      placeholder: 2021.4.0 LTS / Master Branch / tag 2022.3.0
+    validations:
+      required: false
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System
+      description: Operating System 
+      options:
+        - Ubuntu 18.04 (LTS)
+        - Ubuntu 20.04 (LTS)
+        - Windows System
+        - Red Hat Enterprise Linux 8
+        - Android System
+        - Raspbian Stretch OS
+        - macOS Systems for Intel CPU
+        - macOS Systems for Apple Silicon
+        - WebAssembly
+        - Other (Please specify in description)
+    validations:
+      required: true
+  - type: dropdown
+    id: device_use
+    attributes:
+      label: Device Use for Inferencing
+      description: Device Use for Inferencing
+      options:
+        - CPU
+        - GPU
+        - GNA
+        - NCS2 (intel Movidius)
+        - HDDL
+        - AUTO
+        - HETERO
+        - BATCH
+    validations:
+      required: true
+  - type: dropdown
+    id: framework
+    attributes:
+      label: Framework
+      description: Framework use in modal optimization
+      options:
+        - TensorFlow 1
+        - Keras (TensorFlow 2)
+        - Caffe
+        - ONNX
+        - Pytorch
+        - mxnet
+        - Paddle Paddle
+    validations:
+      required: true
+  - type: input
+    id: model_name
+    attributes:
+      label: Model Name  
+      description: Please provide us the link to your model in the description
```

Comment:
Do we have any preferred web storage? It's often a risk to download and open "random" .zip files and so on.